great_expectations/execution_engine/sqlalchemy_execution_engine.py

```diff
@@ -1032,6 +1041,88 @@ def close(self) -> None:
         else:
             self.engine.dispose()
 
+    def _handle_databricks_parameter_limit(self, queries: dict) -> dict:
+        """Handle the Databricks parameter limit.
+
+        Args:
+            queries: The queries to handle.
+
+        Returns:
+            The query with the parameter limit handled.
+        """
+        MAX_PARAMS_PER_QUERY = 256
+        updated_queries: dict[IDDictID, dict] = {}
+        for domain_id, query in queries.items():
```

Comment:
Building the queries then re-building them doesn't seem like a clean/understandable way to do this.

Why not build this into the code above?